internal/printer/pod_test.go

```diff
@@ -261,7 +261,88 @@ func Test_PodListHandlerTerminating(t *testing.T) {
 		"Phase":      component.NewText("Running"),
 		"Status":     component.NewText("Terminating"),
 		"Restarts":   component.NewText("0"),
-		"Age":        component.NewTimestamp(now),
+		"Age":        component.NewTimestamp(creationNow),
+		"_isDeleted": component.NewText("deleted"),
+		"Node":       nodeLink,
+		component.GridActionKey: gridActionsFactory([]component.GridAction{
+			buildObjectDeleteAction(t, pod),
+		}),
+	})
+	addPodTableFilters(expected)
+
+	component.AssertEqual(t, expected, got)
+}
+
+func Test_PodListHandlerTerminatingThreshold(t *testing.T) {
```

Comment:
I may be wrong but it seems like this test does not reach the logic in objectstatus. It may be omittable as discarding the changes still passes the test.

Reply:
If I change the test from `deletionNow := time.Now().Add(time.Minute * -10)` to `deletionNow := time.Now().Add(time.Minute * -4)` it fails, no longer having the expected status.

Reply:
After refactoring the object status tests as suggested, this long form test provided no extra value and I was able to remove it without reducing coverage.